src/components/atoms/SearchBar/index.jsx

```diff
@@ -0,0 +1,131 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+// React PropTypes
+import PropTypes from "prop-types";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBNavbar,
+  MDBNavbarBrand,
+  MDBNavbarNav,
+  MDBNavItem,
+  MDBNavLink,
+  MDBNavbarToggler,
+  MDBCollapse,
+  MDBContainer,
+  MDBDropdown,
+  MDBDropdownItem,
+  MDBDropdownToggle,
+  MDBDropdownMenu,
+  MDBSmoothScroll,
+  MDBIcon,
+  MDBSelect,
+  MDBSelectInput,
+  MDBSelectOption,
+  MDBSelectOptions,
+  MDBBtn,
```

Comment:
Remove unused components.

```suggestion
  MDBSelect,
  MDBSelectInput,
  MDBSelectOption,
  MDBSelectOptions,
```